sns-book/src/domain-name/devnet.md

```diff
@@ -13,17 +13,59 @@ const connection = new Connection("https://api.devnet.solana.com");
 
 Many of the utility and binding functions in the devnet module will require existing devnet domain names. Use the `registerDomainName` binding to register domains on devnet to be used in testing.
 
-```js
-const [, ix] = await devnet.bindings.registerDomainName(
-  connection,
-  "devnet-test-5", // The name of the domain you want to register
-  1_000,
-  publicKey, // PublicKey of fee payer
-  getAssociatedTokenAddressSync(NATIVE_MINT, publicKey, true), // import from @solana/spl-token
-  NATIVE_MINT
-);
+### Notes: 
+
+- The signer must have a WSOL ATA (Associated Token Account) initialized and funded for the transaction to succeed.
```

Comment:
This is only true if you are using wSOL for the devnet registration. More generally, the user must always have the funds he need to pay for the devnet registration whether it's wSOL, USDC etc...